src/sav_system_info.c

```diff
@@ -102,7 +102,7 @@ void Save_SysInfo_RTC_SubField34(SYSINFO_RTC* rtc_info, u32 a1) {
     rtc_info->unk34 -= a1;
 }
 
-void SysInfoRTC_HandleContinueOnNewConsole(SYSINFO_RTC* rtc_info) {
```

Comment:
Pointer notation in this file

Reply:
out of scope - will be addressed in another PR, when I get to the file